README.md

```diff
@@ -2,41 +2,43 @@
+I am a front end developer with leadership skills and over 7 years' experience of developing and delivering projects for a high traffic website. I love shipping new features to users and improving the health of the codebase by addressing tech debt and making decisions around any new tech and libraries used. I have been heavily involved in all aspects of the agile development lifecycle from initial product conversations to deployment and monitoring. 
 
 I am looking for a role that focusses on delivery and maintenance of high quality software projects rather than a team management position.
 
 ## Core Skills
 
-- Web development with React, Redux, Node, ES6, Flow, Typescript, LESS, CSS, CSS in JS, Webpack
+- Web development with React, Redux, Node, Typescript, LESS, CSS, CSS in JS, Webpack
 - Writing meaningful unit, integration and E2E tests using libraries such as Jest, React Testing Library, Enzyme, Mocha, Chai, Webdriverio and Puppeteer
 - Understanding application health and key metrics by implementing Kibana logging, monitoring and dashboards in New Relic One 
-- Responsibility for out of hours infrastructure support and fixing production issues
+- AWS pipeline creation and maintenance and autoscaling setup
 - Strong mentoring and leadership skills, able to build trust with colleagues across the business
 - Pair programming and agile project development
+- Problem solving mentality and positive attitude
 
 ## Experience
 
 ### Trainline (December 2015 - Present)
 
-#### Interim Engineering Team Lead (May 2020 – Present)
-- Managing a full stack team of 8 developers remotely 
-- Working with the Product Owner to plan development work, ensure prioritisation of tech debt and set achievable goals for the team
-
-Achievements
-- I oversaw my team's support for the french site's migration in which we were responsible for migration of user accounts upon login
-- Planned and managed an urgent project as part of a cross team intiative to allow users to amend reservations, in response to new restrictions on some rail services
-
-#### Senior Web Engineer (Apr 2020 – Present)
+#### Senior Web Engineer (Apr 2020 – March 2023)
 - Taking the lead on multiple deliverables requiring collaboration within the team and with other teams 
 - Identifying improvements to testing 
 - Researching and implementing new libraries and new tech (React hooks, React Testing Library)
 - Running front end learning sessions and pairing on TDD with more junior team members
 
 Achievements
-- Helped an API focussed team design new endpoints suitable for our use cases and rewrote parts of the integration test suite to mock the new responses
+- Led two teams to implement white labelling across 4 frontend applications, finding configuration solutions to heavily customised projects
```

Comment:
This sentence read strange to me. "finding configuration solutions to heavily customised projects" particularly